apps/web/lib/api.ts

```diff
@@ -39,20 +39,41 @@ export class ApiClient {
     }
   }
 
-  async post<T>(endpoint: string, body: unknown): Promise<T> {
-    const response = await fetch(`${this.baseUrl}${endpoint}`, {
-      method: "POST",
-      headers: this.getHeaders(),
-      body: JSON.stringify(body),
-    });
-
-    if (!response.ok) {
+  async post(endpoint: string, body: unknown): Promise<Response> {
```

Comment:
What is the problem just extending the post to accept FormData?

`const requestBody = body instanceof FormData ? body : JSON.stringify(body);`